frontend/src/store/bucketStore.ts

```diff
@@ -14,8 +20,19 @@ export const useBucketStore = defineStore('bucket', () => {
   async function load() {
     try {
       loading.value = true;
-      const response = await bucketService.searchForBuckets();
-      buckets.value = response.data;
+
+      const identityId = getKeycloak.value.tokenParsed?.idir_user_guid;
+      if (identityId) {
+        await userStore.searchUsers({ identityId: identityId });
```

Comment:
i wonder if current user's coms `userId` should be saved to a property in the userStore or authStore to avoid having to do a coms user search search

Reply:
That should be the intent. On frontend bootup, assuming you're logged in, you should have in the user store at minimum your current identityId, but no knowledge of the COMS userId. If/when we do invoke the call to hydrate that piece of information, that should be written into the store, and actions that need that piece of information either pull that information from the store since its already there, or force it to do a user call to get it hydrated first. That way we can cache and reduce network calls.